continuationsPluginExample/src/main/scala/examples/Main.scala

```diff
@@ -6,13 +6,8 @@ import continuations.jvm.internal.ContinuationImpl
 @main def main =
 //  val result: Int = program
 //  println(result)
-  val result2 = programSuspendContinuationNoParamResume
-  println(result2)
-  val result3 = programSuspendContinuationNoParamNoSuspendContinuation
-  println(result3)
-  val result4 = programSuspendContinuationNoParamResumeIgnoreResult
-  println(result4)
-  println(programSuspendContinuationParamDependent)
+
+  println(programSuspendContinuationTwoParamsDependent)
```

Comment:
should we not keep the existing ones too?

Reply:
I think we should.

Reply:
Also: there should be a standalone main for `programSuspendContinuationTwoParamsDependent` like we have for the others we just demoed.